components/productions/create/event.vue

```diff
@@ -42,19 +46,28 @@
         default: '',
       },
     },
+    data() {
+      return {
+        period: '',
+      };
+    },
     computed: {
-      period() {
-        const parsedStart = this.parseDate(this.startDate);
-        const parsedEnd = this.parseDate(this.endDate);
-        if (parsedStart === parsedEnd) {
-          return parsedStart;
-        }
-        return `${parsedStart} - ${parsedEnd}`;
+      locale() {
+        return this.$i18n.locale;
+      },
+      eventId() {
+        return parseEventId(this.id);
```

Comment:
I'd suggest parsing the id before passing it as a prop to the components. Then you can avoid having an `id` and `eventId` in these components which is a bit confusing.